core/src/main/java/tc/oc/pgm/command/CancelCommand.java

```diff
@@ -35,6 +39,9 @@ public void cancel(Audience audience, Match match) {
 
     match.getCountdown().cancelAll();
     match.needModule(StartMatchModule.class).setAutoStart(false);
-    audience.sendMessage(translatable("admin.cancelCountdowns", NamedTextColor.GREEN));
+    ChatDispatcher.broadcastAdminChatMessage(
+        translatable(
+            "admin.cancelCountdowns.announce", PlayerComponent.player(sender, NameStyle.FANCY)),
```

Comment:
```suggestion
            "admin.cancelCountdowns.announce", player(sender, NameStyle.FANCY)),
```
We use static imports for components